python/test/measurement/test_average_taxons.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+import unittest.mock
+from test.data_extraction.content_store_helpers import *
```

Comment:
It took me a while to figure out where `content_with_taxons` and `content_without_taxons` were defined. Perhaps we could explicitly declare these two variables from this file rather than everything? I'm hoping we can do something like:

```py
from test.data_extraction.content_store_helpers import content_with_taxons, content_without_taxons
```

Reply:
fixed